q2_feature_table/_subsample_ids.py

```diff
@@ -20,8 +22,14 @@ def subsample_ids(table: biom.Table, subsampling_depth: int,
                          'elements on the desired axis. The maximum depth '
                          'is: %d.' % len(table.ids()))
 
+    # Generate a random seed if seed is None
+    if seed == "random":
```

Comment:
You can use the built-in [`random.seed` function](https://docs.python.org/3/library/random.html#random.seed) for this.